mob.go

```diff
@@ -751,6 +751,17 @@ func hasUncommittedChanges() bool {
 	return !isNothingToCommit()
 }
 
+func hasUnpushedCommits(branch string, configuration Configuration) bool {
+	countOutput := silentgit(
+		"rev-list", "--count", "refs/heads/"+branch+"..."+"refs/remotes/"+configuration.RemoteName+"/"+branch,
+	)
+	unpushedCount, err := strconv.Atoi(strings.TrimSpace(countOutput))
+	if err != nil {
+		panic(err)
```

Comment:
an info log about the number of unclpushed commits found would be nice

also in the lastest commit in main the silentgit method now always trims.

Reply:
k, I'll rebase against upstream in a moment.